src/NUnitConsole/nunit3-console/nunit3-console.csproj

```diff
@@ -3,7 +3,7 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>NUnit.ConsoleRunner</RootNamespace>
     <AssemblyName>nunit3-console</AssemblyName>
-    <TargetFrameworks>net20;netcoreapp2.1</TargetFrameworks>
```

Comment:
The console is as-yet unreleased, but will now target .NET Core 3.1 as a minimum. With the long-term aim of making it self-contained anyway, I'm not too worried about this.